cmd/adsysd/main.go

```diff
@@ -22,6 +22,7 @@ import (
 
 func main() {
 	var a app
+	i18n.InitI18nDomain(consts.TEXTDOMAIN)
```

Comment:
I think the a declaration is for the daemon below. So I would just move that one up. It’s not C89 :)